docs/CONTRIBUTING.md

```diff
@@ -37,3 +38,8 @@ if (true)
   // your code here
 }
 ```
+
+## Building (only useful for the maintainer, don't do it)
+From the project's `src/` directory, execute `make VERSION=testing`. This will create an XPI file in the project's `bin/testing/` directory that you can install in your browser.
+
+(To be confirmed) Note that Firefox will not accept unsigned extensions unless you deactivate `xpinstall.signatures.required` in `about:config`.
```

Comment:
What does "to be confirmed" mean?

Reply:
I don't need to deactivate `xpinstall.signatures.required` for testing in my Firefox 52 ESR.
Maybe I'm missing something. Related documentation on MDN https://developer.mozilla.org/en-US/Add-ons/Distribution


Thanks for the review, I'm pushing a commit with your suggestions :)